public/javascripts/SVLabel/src/SVLabel/util/Utilities.js

```diff
@@ -125,13 +125,14 @@ if(typeof JSON!=="object"){JSON={}}(function(){"use strict";function f(e){return
 // Get what browser the user is using.
 // This code was taken from an answer in the following SO page:
 // http://stackoverflow.com/questions/3303858/distinguish-chrome-from-safari-using-jquery-browser
+// addendum 6-21-2017: chrome
```

Comment:
Whoops, definitely thought I had a longer comment than that. Just wanted to note that I added code for recognizing Chrome browsers on iOS and that this was distinct from the stackoverflow link mentioned. Obviously that wasn't clear. 😅